.travis.yml

```diff
@@ -7,7 +7,7 @@ notifications:
 # Run latest 3.x and 2.x next, to get quick compatibility results.
 # Then run the remainder.
 python:
-  - "pypy"
+  - "pypy-5.3.1"
```

Comment:
@wiredfool  but the test not related to adding CTL support. Is it fine to edit that test or you may  edit later?